### ISSUE TITLE
Hiding the keyField Column

### ISSUE THREAD

**Issue by nk-gears** [mjakal/react-async-table#1]
How to hide the KeyField Column ?.

Also, for the Multiple delete, there is a bug if the id has a string value. There is a code parseInt which corrupts the values selected.

**Comment by mjakal** [comment]
You can remove it from columns array, see the example below.

```
const columns = [
  /*
  {
    dataField: 'id',
    text: 'ID'
  },
  */
  {
    dataField: 'userId',
    text: 'User ID',
  },
  {
    dataField: 'title',
    text: 'Task',
    tooltip: 'Example Tooltip',
    sort: true,
    sortOrder: 'asc' // values: asc|desc
  },
  {
    dataField: 'completed',
    text: 'Status',
    formatedField: ExampleFormatedField
  }
];
```
I'll fix multiple delete issue in v 0.0.31.

**Comment by nk-gears** [comment]
thanks. I created a separate fork and updating relevant changes. Will send you a separate PR later
